fastscroll/src/main/java/com/l4digital/fastscroll/FastScroller.java

```diff
@@ -728,10 +722,11 @@ private void layout(Context context, AttributeSet attrs, Size size) {
 
         bubbleSize = size;
 
-        @ColorInt int bubbleColor = Color.GRAY;
-        @ColorInt int handleColor = Color.DKGRAY;
+        final int accent = getColorAttr(context, R.attr.colorAccent);
+        @ColorInt int bubbleColor = accent;
+        @ColorInt int handleColor = accent;
         @ColorInt int trackColor = Color.LTGRAY;
-        @ColorInt int textColor = Color.WHITE;
+        @ColorInt int textColor = getColorAttr(context, android.R.attr.windowBackground);
```

Comment:
These changes are not good. They assume the app has the color value defined (likely, but not certain), and since the library doesn't define this color it would be problematic for consumers who also do not. If consumers want to use the accent colors, they can easily set those using the provided xml attributes.